app/controllers/presentations_controller.rb

```diff
@@ -5,10 +5,12 @@ def show
 
   def new
     @presentation = Presentation.new
+    authorize @presentation
```

Comment:
Looks like we can do an `around_filter`. We would have to throw yield statements in place of all the authorize statements so I don't think it really cleans any thing up. Not really feeling that approach so much.